contracts/Makefile

```diff
@@ -9,7 +9,7 @@ test: $(SUBDIRS) ## Run all the tests in the subfolder
 
 wasm: $(SUBDIRS) ## Generate the WASM for all the contracts
 
-$(SUBDIRS):
-	$(MAKE) -C $@ $(MAKECMDGOALS)
+#$(SUBDIRS):
+#	$(MAKE) -C $@ $(MAKECMDGOALS)
```

Comment:
Why is this commented out?

Reply:
For now the contracts are not updated with the latest canonical. They will be updated in a subsequent PR